crates/nu-parser/src/parse.rs

```diff
@@ -1350,11 +1349,11 @@ fn classify_pipeline(
 
             commands.push(ClassifiedCommand::Internal(InternalCommand {
                 name: "run_external".to_string(),
-                name_span,
+                name_span: Span::unknown(),
```

Comment:
I think this may break error reporting. I may have to undo this. I originally did this to address an issue I had in the completion engine, but it may no longer be necessary.

Reply:
It does, fixing.